dataland-frontend/tests/e2e/specs/esg-questionnaire/DataIntegrity.ts

```diff
@@ -46,34 +48,36 @@ describeIf(
         getKeycloakToken(admin_name, admin_pw).then((token: string) => {
           return uploadCompanyViaApi(token, generateDummyCompanyInformation(testCompanyNameEsgQuestionnaire)).then(
             (storedCompany) => {
-              return uploadGenericFrameworkData(
-                token,
-                storedCompany.companyId,
-                '2021',
-                esgQuestionnaireFixtureForTest.t,
-                (config) =>
-                  getBasePublicFrameworkDefinition(DataTypeEnum.EsgQuestionnaire)!.getPublicFrameworkApiClient(config)
-              ).then((dataMetaInformation) => {
-                cy.intercept(`**/api/data/${DataTypeEnum.EsgQuestionnaire}/${dataMetaInformation.dataId}`).as(
-                  'fetchDataForPrefill'
-                );
-                cy.visitAndCheckAppMount(
-                  '/companies/' +
-                    storedCompany.companyId +
-                    '/frameworks/' +
-                    DataTypeEnum.EsgQuestionnaire +
-                    '/upload?templateDataId=' +
-                    dataMetaInformation.dataId
-                );
-                cy.wait('@fetchDataForPrefill', { timeout: Cypress.env('medium_timeout_in_ms') as number });
-                cy.get('h1').should('contain', testCompanyNameEsgQuestionnaire);
-                cy.intercept({
-                  url: `**/api/data/${DataTypeEnum.EsgQuestionnaire}`,
-                  times: 1,
-                }).as('postCompanyAssociatedData');
-                submitButton.clickButton();
-                cy.wait('@postCompanyAssociatedData', { timeout: Cypress.env('medium_timeout_in_ms') as number }).then(
-                  (postInterception) => {
+              return assignCompanyOwnershipToDatalandAdmin(token, storedCompany.companyId).then(() => {
```

Comment:
Warum brauchen wir das?

Reply:
Idee hierzug um den company owner status auch zu nutzen. Beim Besuchen der Viewpage im DataIntegrity Test sollte geprüft werden, dass die Review-Buttons (Accept, Decline) nicht angezeigt werden, wenn der Datensatz als Company Owner hochgeladen wurde. Damit stellen wir dann sicher, dass die Funktionalität auch wie gewollt funktioniert. 
Heißt die Tests müssten mit so einem kleinen Check jeweils erweitert werden.

Reply:
check added to all dataintegrity tests

Reply:
ok, passt